### PR TITLE
Fix missing ollama import for generation

### PR DESCRIPTION
**Pull request by Guillaume351**
ollama generation does not work anymore due to this missing import

### CODE REVIEW

**Review by rajakumar05032000 — COMMENTED** [review]
You don't have to import generate.

client object has been created in the code by supplying host in line 9.

Use the same object for inference in line 25.

File: src/llm/ollama_client.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def list_models():
 
     def inference(self, model_id: str, prompt: str) -> str:
         try:
-            response = ollama.generate(model=model_id, prompt=prompt.strip())
+            response = generate(model=model_id, prompt=prompt.strip())
```
Comment (rajakumar05032000) [inline src/llm/ollama_client.py:25]: it should be:
response = client.generate(model=model_id, prompt=prompt.strip())

Comment (Guillaume351) [reply src/llm/ollama_client.py]: Thank you. I've made the modification

**Review by rajakumar05032000 — APPROVED** [review]

**Comment by ARajgor** [comment]
#24

**Pull request closed without merging**